src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+
+    private double radius;
+
+    public Circle(Color color, double radius) {
+        super(color);
+        this.radius = radius;
```

Comment:
Creating a figure, don't pass expressions in the constructor. It's better to create separate variables and pass them on for better code readability. This can be handled in the `FigureSupplier` class where you should generate the random values and pass them to the constructor.